pkgs/applications/audio/mopidy/ytmusic.nix

```diff
@@ -16,10 +16,14 @@ python3Packages.buildPythonApplication rec {
     python3Packages.pytube
   ];
 
-  doCheck = false;
```

Comment:
> Ran 0 tests in 0.000s

That line served a purpose and should be restored.. It disabled tests, since the package has none.

Reply:
Also we could add a pytonImportsCheck regardless of tests.